src/flb_output.c

```diff
@@ -587,7 +587,7 @@ int flb_output_init(struct flb_config *config)
 #endif
 
 #ifdef FLB_HAVE_TLS
-        if (ins->flags & FLB_IO_TLS) {
+        if (ins->use_tls == FLB_TRUE) {
```

Comment:
Seems out of sync with the cleanup logic:

```
#ifdef FLB_HAVE_TLS
    if (ins->flags & FLB_IO_TLS) {
        if (ins->tls.context) {
            flb_tls_context_destroy(ins->tls.context);
        }
    }
#endif
```

Reply:
Thanks for pointing out. I have corrected it.